src/SDKs/Billing/Management.Billing/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Billing Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Billing.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
Please do not update the `AssemblyVersion`

Reply:
Reverted